### PR TITLE
Add dmesg error filter and update board table

### PR DESCRIPTION
**Pull request by tagoylo**
Signed-off-by: Trecia Agoylo <trecia.agoylo@analog.com>

Add dmesg error filter
Update board_table.yaml for zed-adrv9002

### CODE REVIEW

File: nebula/network.py [hunk 1]
```diff
@@ -262,9 +263,25 @@ def check_dmesg(self, error_on_warnings=False):
...
             error_log = f.readlines()
+            errors = []
+            for err in error_log:
+                error = err.split("]", 1)[1].strip()
+                errors.append(error+"\n")
 
-        if len(error_log) > 0:
-            logging.info("Errors found in dmesg logs")
+        path = pathlib.Path(__file__).parent.absolute()
+        res = os.path.join(path, "resources", "err_rejects.log")
```
Comment (kimpaller) [inline nebula/network.py]: I am not sure non-python files like text files., would be included during installation (python setup.py install) without specifying it first in  setup.py like this one., https://github.com/tfcollins/nebula/blob/8113c1e02ac1f45d5320477b45bb73f9ef3dbe0c/setup.py#L33., i think you also need to include err_rejects.log /txt  on that line.

File: nebula/network.py [hunk 2]
```diff
@@ -262,9 +263,25 @@ def check_dmesg(self, error_on_warnings=False):
             warn_log = f.readlines()
         with open("dmesg_err.log", "r") as f:
             error_log = f.readlines()
+            errors = []
+            for err in error_log:
+                error = err.split("]", 1)[1].strip()
+                errors.append(error+"\n")
```
Comment (kimpaller) [inline nebula/network.py:269]: check if '\n' is necessary., i'm not sure if 
`error_rejects = f.readlines()`
will include \n on each line

Comment (tagoylo) [reply nebula/network.py]: used lstrip instead of strip to remove only the leading whitespace and in order to remove \n. a226d3a

**Review by kimpaller — COMMENTED** [review]
No more additional review comments. This is good.

**Pull request merged**